OWNERS

```diff
@@ -1,6 +1,5 @@
 approvers:
 - arschles
-- bmelville
 - duglin
 - jpeeler
 - krancour
```

Comment:
I am following up with Kent on if we can remove him from this list so that he's not pinged on reviews anymore.

Reply:
Confirmed, he's cool with being removed

Reply:
I'm cool with removing in this PR or as a follow-up. Your choice